webapp/sources/ldap-inventory/inventory-api/src/main/scala/com/normation/inventory/domain/NodeInventory.scala

```diff
@@ -365,7 +365,21 @@ final object SoftwareUpdateKind {
 
   def parse(value: String) = all.find( _.name == value.toLowerCase()).getOrElse(Other(value))
 }
+sealed trait SoftwareUpdateSeverity {
+  def name: String
+}
+
+final object SoftwareUpdateSeverity {
+  final case object Low                  extends SoftwareUpdateSeverity { val name = "low" }
+  final case object Moderate             extends SoftwareUpdateSeverity { val name = "moderate" }
+  final case object High                 extends SoftwareUpdateSeverity { val name = "high" }
+  final case object Critical             extends SoftwareUpdateSeverity { val name = "critical" }
+  final case class  Other(value: String) extends SoftwareUpdateSeverity { val name = "other" }
```

Comment:
I'd add a "none" value

Reply:
Since it's not mandatory, I though the case none was redondant

Reply:
indeed!